### ISSUE TITLE
Which version of Python does IfcOpenShell-Python currently support?

### ISSUE THREAD

**Issue by keleisi2001** [IfcOpenShell/IfcOpenShell#1970]
My anaconda version corresponds to Python 3.9.7, and there are many conflicts during the installation process.

**Comment by aothms** [comment]
Which channel did you use?

**Comment by keleisi2001** [comment]
> Which channel did you use?

My previous computer was using Python 3.7.4 and it ran very smoothly. The new computer is using Python 3.9.7 and these conflicts appear (as in the screenshot)
![screenshot](https://user-images.githubusercontent.com/55388367/148629451-60fde87e-dedd-4217-9184-858a6148c864.png)

**Comment by keleisi2001** [comment]
I reinstalled anaconda3 based on Python 3.7.6, and IfcOpenShell has been successfully installed. According to my own attempts, Python 3.8 and above will have some conflicts, I guess this is caused by other environments such as OCC required by IfcOpenShell. As far as I understand OCC 7.4.0 was updated in summer 2019, while Python 3.8.0 was updated in October 2019 and 3.9.0 was released in October 2020.

In short, Python 3.7.6 (corresponding to anaconda 2020. 02) is perfectly compatible with the current IfcOpenShell.

**Comment by aothms** [comment]
If it works then it works :) but I must say you use a lot of channels just to install ifcopenshell. conda-forge should be sufficient and that might also reduce the risk of conflicts perhaps (not entirely sure how that works).

**Comment by Moult** [comment]
This can probably be closed, but it is worth mentioning for future readers that 3.7, 3.8, and 3.9 are all supported.

**Comment by brunopostle** [comment]
> it is worth mentioning for future readers that 3.7, 3.8, and 3.9 are all supported

Also 3.10 works ok here.
